cmd/gcp-controller-manager/app/node_annotater.go

```diff
@@ -156,29 +174,27 @@ func (na *nodeAnnotater) sync(key string) {
 	}
 }
 
-func (na *nodeAnnotater) getExternalID(nodeUrl string) (string, error) {
+type annotater struct {
```

Comment:
how about `type annotator func(*core.Node, *compute.Instance) bool` ?

Reply:
I think we will want to add more fields (e.g. declare what change should trigger this annotator).